package.json

```diff
@@ -37,7 +37,7 @@
     "react-hook-form": "^7.41.1",
     "react-hook-form-mui": "^5.12.0",
     "react-icons": "^4.9.0",
-    "react-share": "^5.0.2",
+    "react-share": "^4.4.1",
```

Comment:
Can you please remove the package lock and package files both from the PR so that we may keep the updated library as this was part of the share button changes.